normandy/recipes/migrations/0024_auto_20160422_0541.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2016-04-22 05:41
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('recipes', '0023_auto_20160324_2333'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='recipe',
+            name='countries',
+        ),
+        migrations.RemoveField(
+            model_name='recipe',
+            name='end_time',
+        ),
+        migrations.RemoveField(
+            model_name='recipe',
+            name='locales',
+        ),
+        migrations.RemoveField(
+            model_name='recipe',
+            name='release_channels',
+        ),
+        migrations.RemoveField(
+            model_name='recipe',
+            name='sample_rate',
+        ),
+        migrations.RemoveField(
+            model_name='recipe',
+            name='start_time',
+        ),
+        migrations.AddField(
+            model_name='recipe',
+            name='filter_expression',
+            field=models.TextField(default='(1 + 1 == 2)'),
```

Comment:
`'(1 + 1 == 2)'` is a weird default. I think `''` would be better. The addon, at least, interprets an empty `filter_expression` as matching anything (just like an empty matcher did before).